ext/page_log/palm/palm.c

```diff
@@ -333,6 +333,24 @@ palm_resize_item(WT_ITEM *item, size_t new_size)
     return (0);
 }
 
+/*
+ * palm_init_context --
+ *     Initialize a context in a standard way.
+ */
+static void
+palm_init_context(PALM *palm, PALM_KV_CONTEXT *context)
+{
+    memset(context, 0, sizeof(*context));
+
+    /*
+     * To get more testing variation, We could call palm_compute_delay_us to randomize this number.
+     * If we do so, we need to make sure items are materialized in the same order they are written.
+     * So when setting PAGE_KEY.timestamp_materialized_us, we'd need to make each value set was
+     * monotonically increasing.
+     */
+    context->materialization_delay_us = palm->materialization_delay_ms;
```

Comment:
"us" and "ms" are different units... did you forget to multiply by 1000?

Reply:
Ah - that's it, I'll fix.